peer/network.go

```diff
@@ -38,6 +38,7 @@ var (
 	_                     common.AppHandler    = &network{}
 )
 
+// TODO remove this interface
```

Comment:
If we need this interface it should be defined in the consuming package `plugin`